packages/shared/src/components/squads/Details.tsx

```diff
@@ -165,7 +173,41 @@ export function SquadDetails({
   };
 
   return (
-    <>
+    <ConditionalWrapper
+      condition={isMobile}
```

Comment:
Should the condition not be `isMobile && createMode`?

Reply:
On Edit mode, there should be a "Save" button.

Reply:
But still in a form wrapper?